File: src/view/docs/demos/App.Site.Index.cs
PhalApiClientResponse response = PhalApiClient.create()
  .withHost("http://demo.phalapi.net/")
  .withService("Default.Index")
  .withparamsList("name", "dogstar")
  .withTimeout(3000)
  .request();

Log.v("response ret", response.ret + "");
Log.v("response data", response.data);
Log.v("response msg", response.msg);

File: src/view/docs/demos/App.Site.Index.go
func main() {
	rs, err := PhalApiClient.NewRequest().
		WithHost(`{url}`).
		WithService("{s}").
		WithParams(url.Values{}).
		Get()
	if err != nil {
		fmt.Println(err.Error())
	} else {
		fmt.Println("code------------------------", rs.Code)
		fmt.Println("data------------------------", rs.Data)
		fmt.Println("msg------------------------", rs.Msg)
	}
}